docs/docusaurus.config.js

```diff
@@ -0,0 +1,148 @@
+  projectName: "testkube", // Usually your repo name.
+
+  // Even if you don't use internalization, you can use this field to set useful
+  // metadata like html lang. For example, if your site is Chinese, you may want
+  // to replace "en" with "zh-Hans".
+  i18n: {
+    defaultLocale: "en",
+    locales: ["en"],
+  },
+
+  presets: [
+    [
+      "classic",
+      /** @type {import('@docusaurus/preset-classic').Options} */
+      ({
+        docs: {
+          routeBasePath: "/",
+          sidebarPath: require.resolve("./sidebars.js"),
+          // Please change this to your repo.
+          // Remove this to remove the "edit this page" links.
+          editUrl: "https://github.com/kubeshop/testkube/docs",
+        },
+        blog: false,
+        theme: {
+          customCss: require.resolve("./src/css/custom.css"),
+        },
+        googleAnalytics: {
+          trackingID: "UA-204665550-6",
+        },
+      }),
+    ],
+    [
+      "redocusaurus",
+      {
+        // Plugin Options for loading OpenAPI files
+        specs: [
+          {
+            spec: "https://raw.githubusercontent.com/kubeshop/testkube/main/api/v1/testkube.yaml",
+            route: "/openapi",
+          },
+        ],
+        theme: {
+          primaryColor: "#818cf8",
+        },
+      },
+    ],
+  ],
+
+  themeConfig:
+    /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
+    ({
+      navbar: {
+        logo: {
+          alt: "Testkube",
+          src: "img/logo.svg",
+          href: "/",
+        },
+        items: [
+          {
+            type: "html",
+            position: "right",
+            value: `<iframe src="https://ghbtns.com/github-btn.html?user=kubeshop&repo=testkube&type=star&count=true&color=dark" style='margin-top: 6px' frameborder="0" scrolling="0" width="110" height="20" title="GitHub"></iframe>`,
+          },
+          {
+            type: "search",
+            position: "left",
+          },
+        ],
+      },
+
+      footer: {
+        style: "dark",
+        links: [
+          {
+            title: "Community",
+            items: [
+              {
+                label: "Discord",
+                href: "https://discord.com/invite/6zupCZFQbe",
+              },
+              {
+                label: "Twitter",
+                href: "https://twitter.com/Testkube_io",
+              },
+              {
+                label: "GitHub",
+                href: "https://github.com/kubeshop/testkube",
+              },
+            ],
+          },
+        ],
+        copyright: `Copyright © ${new Date().getFullYear()} Kubeshop.`,
+      },
+      prism: {
+        theme: lightCodeTheme,
+        darkTheme: darkCodeTheme,
+      },
+      algolia: {
+        // The application ID provided by Algolia
+        appId: "QRREOKFLDE",
+
+        // Public API key: it is safe to commit it
+        apiKey: "97a76158bf582346aa0c2605cbc593f6",
```

Comment:
Should it be stored in github secrets?

Reply:
it's a public key – so no need for a secret :) 
but we could store it as an ENV var (not sure about the benefit of this since it's not shared across any other instance)